src/mtool/api/Dockerfile-flask

```diff
@@ -8,6 +8,8 @@ FROM python:latest
 ENV APP /app
 ENV PYTHONUNBUFFERED 1
 ENV PYTHONIOENCODING UTF-8
+ENV http_proxy http://12.26.204.100.8080/
```

Comment:
I think it would be good to change the proxy environment variable to receive it through the docker-compose file.

Reply:
I agree with your opinion. Move it to compose file